onnxruntime/core/providers/iree/compiler/torch-mlir-import-onnx/OnnxImporter.cpp

```diff
@@ -231,6 +231,7 @@ const onnx::TypeProto *GraphInfo::FindTypeProtoForName(std::string_view name) {
   // Node outputs don't typically have type information, but shape inference
   // will associate them in the value_info. If not there, it may be a
   // graph output, which must have type information.
+#if 0
```

Comment:
Just remove dead code? If you want to preserve context for something that should be added back later, comment the code block with a descriptive todo like "Reconsider how to ...  Vs the commented approach"